.github/workflows/ci.yml

```diff
@@ -126,6 +126,7 @@ jobs:
         - 5432:5432
     env:
       TEST_DATABASE_URL: postgresql://postgres@localhost:5432/report_test
+      DATABASE_URL: postgresql://postgres@localhost:5432/report_test
```

Comment:
Might be nice to start off documenting our env vars in this project like we've done in some of the newer ones.